setup.py

```diff
@@ -35,6 +35,12 @@
             'pytest-mock',
             'Sphinx',
             'sphinx_rtd_theme'
-        ]
+        ],
+        'testbed': [
+            'six>=1.10.0,<2.0',
+            'futures',
```

Comment:
I was under impression futures is not available in py3

Reply:
Technically it should not be installed for py3, but since the stdlib takes precedence in modules resolution, it should not be harmful / wrong in our `testbed`. Conditional requires are available as follow:
```python
setup(
    ...
    extras_require={
        ':python_version == "2.7"': ['futures']
    }
)
```
as described in the `futures` [package](https://pypi.python.org/pypi/futures)